### PR TITLE
chore: define multiple exports

### PR DESCRIPTION
**Pull request by chrishelgert**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +33,7 @@
   },
   "scripts": {
     "start": "vite build --watch",
-    "build": "tsc && vite build",
```
Comment (aodhagan-cf) [inline package.json:24]: Don't actually know if we need tsc here as the plugin generates the types for us 🤔

File: src/index.ts [hunk 2]
```diff
@@ -4,6 +4,8 @@ import { LiveUpdates } from './live-updates';
 import { Entity, LivePreviewProps, SubscribeCallback, TagAttributes } from './types';
 import { sendMessageToEditor } from './utils';
 
+export * from './react';
```
Comment (aodhagan-cf) [inline src/index.ts:7]: I've included react in the index bundle... would love to not do that

File: package.json [hunk 3]
```diff
@@ -6,12 +6,28 @@
...
   "files": [
-    "dist"
+    "dist",
+    "react",
+    "style.css"
   ],
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
```
Comment (chrishelgert) [inline package.json:20]: we could add the "types": "./dist/index.d.ts", as it should now  work since ts@4.9

Comment (aodhagan-cf) [reply package.json:20]: Does that mean that all of our customers need to be on that version of ts?

Comment (chrishelgert) [reply package.json:20]: mh you're right that would be the requirement

**Comment by aodhagan-cf** [comment]
Hey @chrishelgert, is anything else needed on this PR?

**Comment by chrishelgert** [comment]
> Hey @chrishelgert, is anything else needed on this PR?

@aodhagan-cf no should be ready to ship 🚢

**Review by YvesRijckaert — APPROVED** [review]
Let's create a follow up to update the live-preview package in the marketing template + REST template?

**Comment by chrishelgert** [comment]
> Let's create a follow up to update the live-preview package in the marketing template + REST template?

Can do this directly once it is published :)

**Review by 2wce — APPROVED** [review]

**Pull request merged**